### DISCUSSION TITLE
What is the same method as pako.deflate?

### DISCUSSION THREAD

**Discussion by zzmingo** [101arrowz/fflate#28] [Q&A] [answered]
I know that the same as pako.deflateRaw is fflate.deflate, what is the same method as pako.deflate?

**Comment by 101arrowz** [comment, marked as answer]
There's not much continuity among the different compression libraries, unfortunately, but I didn't want to conform to a misnomer for the actual compression algorithm being used. `fflate`'s naming convention is correct; the other libraries are wrong. That said, here's a direct mapping:

`pako.deflateRaw` = `zlib.deflateRawSync` = `fflate.deflateSync`
`zlib.deflateRaw` = `fflate.deflate`
`pako.deflate` = `zlib.deflateSync` = `fflate.zlibSync`
`zlib.deflate` = `fflate.zlib`

I'll add a full list to the wiki.

**Reply by ThaUnknown** [reply]
@101arrowz any ETA on this table? I'd like to use this as a polyfill for `zlib` and I'm not certain on what to use instead of zlib's `inflateSync`

**Reply by 101arrowz** [reply]
That would just be `unzlibSync`. I forgot entirely about this but I will try to remember to add it when I get a chance.

**Reply by ThaUnknown** [reply]
> unzlibSync

ah but that outs uint8 not string :/

**Reply by 101arrowz** [reply]
You can convert to and from string with `strFromU8` and `strToU8`

**Comment by yisibl** [comment]
@101arrowz Can you add a guide to migrating from pako in Readme? I was confused when I tried to migrate to fflate.
